content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -222,50 +164,123 @@ The drawbacks of ORMs are not always apparent when you start using them. This se
+
+As an example, here's a Prisma schema for a blog:
 
 ```prisma
-model User {
-  id        Int      @default(autoincrement()) @id
-  birthDate DateTime
-  email     String   @unique
-  firstName String?
-  lastName  String?
-  posts     Post[]
+datasource db {
+  provider = "postgres"
+  url      = "postgresql://janedoe:janedoe@localhost:5432/hello-prisma"
+}
+
+generator client {
+  provider = "prisma-client-js"
 }
 
 model Post {
-  content   String?
-  post_id   Int     @default(autoincrement()) @id
+  id        Int     @id @default(autoincrement())
   title     String
-  author_id User
+  content   String? @map("post_content")
+  published Boolean @default(false)
+  author    User?   @relation(fields: [authorId], references: [id])
+  authorId  Int?
+}
+
+model User {
+  id    Int     @id @default(autoincrement())
+  email String  @unique
+  name  String?
+  posts Post[]
 }
 ```
 
-Prima schema's declarative nature is concise compared to imperative model definitions in ORMs.
+Here's a break down of the example above:
+
+- The `datasource` block defines the connection to the database.
+- The `generator` block tells Prisma to generate the client for TypeScript and Node.js.
+- The `Post` and `User` models map to database tables.
+- The two models have a _1-n_ relation where each `User` can have many related `Post`s.
+- Each field in the models has a type, e.g. the `id` has the type `Int`.
+- Fields may contain field attributes to define:
+  - Primary keys with the `@id` attribute.
+  - Unique keys with the `@unique` attribute.
+  - Default values with the `@default` attribute.
+  - Mapping between table columns and Prisma Client fields with the `@map` attribute, e.g., the `content` field maps (which will be accessible in Prisma Client) maps to the `post_content` database column.
+
+The `User` / `Post` relation can be visualized with the following diagram:
+
+![1-n relation between User and Post](https://imgur.com/lidq1Xy.png)
 
-The workflow is slightly different to ORMs. You can use Prisma when building new applications from scratch or adopt it incrementally:
+At a Prisma level, the `User` / `Post` relation is made up of:
+
+- The scalar `authorId` field, which is referenced by the `@relation` attribute. This field exists in the database table – it is the foreign key that connects Post and User.
+- The two relation fields: `author` and `posts` **do not exist** in the database table. Relation fields define connections between models at the Prisma level and do not exist in the database. These fields are used to access relations with Prisma Client.
+
+Prima schema's declarative nature is concise and allows defining tables, relations, table constraints, and mappings between the table columns and their representation in Prisma Client (like the `content` field).
```

Comment:
This is a bit of a repetition of above, but might be useful to get the point across of course.

Reply:
Shortened it to:

> Prima schema's declarative nature is concise and allows defining the database schema and corresponding representation in Prisma Client.